### PR TITLE
serenity: Add myself and the mailing list to the cc list.

### PR DESCRIPTION
**Pull request by bgianfo**
CC: @awesomekling @linusg

### CODE REVIEW

**Comment by bgianfo** [comment]
Is this not going to work (the mailing list subscription) since you still need your email registered to view the links included in the report?

**Comment by awesomekling** [comment]
No idea, but I never log in fwiw, I just view the public reports on the oss-fuzz bug tracker. You can see the bugs and download the repro cases without logging in.

File: projects/serenity/project.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ auto_ccs:
   - "ali.mpfard@gmail.com"
   - "luke.wilde@live.co.uk"
   - "bugaevc@serenityos.org"
+  - "b.gianfo@gmail.com"
+  - "~awesomekling/serenityos-dev@lists.sr.ht"
```
Comment (jonathanmetzman) [inline projects/serenity/project.yaml:11]: Just confirming: Is "~awesomekling/serenityos-dev@lists.sr.ht" an email address? it looks very weird to me.

Comment (bgianfo) [reply projects/serenity/project.yaml:11]: Yes, that's the format of mailing lists hosted on: http://sr.ht 
See: https://lists.sr.ht/~awesomekling/serenityos-dev

**Pull request merged**